server/internal/repositories/servicestest/internal/suite/build.go

```diff
@@ -0,0 +1,78 @@
+package suite
+
+import (
+	"log"
+	"os"
+
+	fake "github.com/brianvoe/gofakeit/v6"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+
+	"github.com/quocbang/data-flow-sync/server/internal/repositories"
+	"github.com/quocbang/data-flow-sync/server/internal/repositories/orm/models"
+)
+
+type SuiteConfig struct {
+	*suite.Suite
+	DB   *gorm.DB
+	Repo repositories.Repositories
+
+	relativeModels       []models.Models
+	clearDataForEachTest bool
+}
+
+type NewSuiteParameters struct {
+	// just do test or clear data in those models also data in database.
+	RelativeModels []models.Models
+	// clear all data in RelativeModels.
+	ClearDataForEachTest bool
+}
+
+func NewSuiteTest(params NewSuiteParameters) *SuiteConfig {
+	return &SuiteConfig{
+		relativeModels:       params.RelativeModels,
+		Suite:                &suite.Suite{},
+		clearDataForEachTest: params.ClearDataForEachTest,
+	}
+}
+
+var logger *zap.Logger
+
+func (s *SuiteConfig) SetupSuite() {
+	ranDomSeed := fake.UUID()
```

Comment:
I think you didn't use the `ranDomSeed`

Reply:
something wrong?

Reply:
if you want to use the seed you must call `fake.Seed(randomSeed)` and random seed should be int64
btw the name should be random seed

Reply:
are you sure? `gofakeit.Seed` didn't return anything, and we need a random number or string to make sure each test is each different

![image](https://github.com/quocbang/data-flow-sync/assets/89030755/9cb0b385-edd4-47f9-8467-e4751f864a32)